client/my-sites/themes/themes-selection.jsx

```diff
@@ -119,31 +123,32 @@ const ThemesSelection = React.createClass( {
 	},
 
 	render() {
-		const site = this.props.selectedSite;
+		const { selectedSite: site } = this.props;
```

Comment:
Nothing wrong with this, just amuses me when the 'shorthand' is longer than the old line :)